object.go

```diff
@@ -123,47 +122,7 @@ func (o *Object) createCall(ctx context.Context, method string, flags Flags, ch
 	if len(args) > 0 {
 		msg.Headers[FieldSignature] = MakeVariant(SignatureOf(args...))
 	}
-	if msg.Flags&FlagNoReplyExpected == 0 {
```

Comment:
Where is this logic relocated?

Reply:
There is equivalent logic within the `Conn.send()` method (the guts of `SendWithContext`):

```
	if msg.Type == TypeMethodCall && msg.Flags&FlagNoReplyExpected == 0 {
		...
```

Reply:
Ah, ok